### DISCUSSION TITLE
Add a view with functionality like web/desktop's "Notifications" pane, filterable and searchable

### DISCUSSION THREAD

**Discussion by HarHarLinks** [element-hq/element-meta#96] [Ideas]
### Your use case

![image](https://user-images.githubusercontent.com/2803622/146937421-5a16e2bc-5af5-4aa5-865b-9250f49e2f30.png)

(This was originally an android issue, but is applicable to all platforms)

### Additional context

vector-im/element-android#3822 is a soft variant that can't replace this, and should not be disregarded but maybe added as well

**Comment by daniellekirkwood** [comment]
Thanks for your feedback @HarHarLinks. 
I think this is a really interesting idea, can you tell me more information about why you would want to see your notifications/unreads in this way? Do the current notification badges lack important info for you?

**Comment by ShadowJonathan** [comment]
Can give some of my own insights; the Notification pane gives a clear "oh this is what I was pinged for" history, which then basically gives a sub-feed of "the important stuff". Notifications themselves somewhat fill this too, but this pane leans heavily on the "history" side of things, so that one can scroll up to see where they got mentioned 4 hours ago, when they were in the middle of a call and accidentally clicked the notification.

**Comment by HarHarLinks** [comment]
What Jonathan said.

Most essentially I would like to revisit a message that pinged me earlier easily. I don't know of any way to do this on android besides scrolling manually.

The other essential idea is to navigate between multiple unread pings in a room.

**Comment by ShadowJonathan** [comment]
For the latter, I would recommend telegram's design, which has a "go to the next mention" button above the "go to the bottom of the page" button.

(I could grab visuals for this on request)

**Comment by HarHarLinks** [comment]
> (I could grab visuals for this on request)

probably vector-im/element-android#3822?

However my idea is instead of (supposedly, I don't have telegram) jumping to the last ping, give users agency to select them from a list instead of having to "click through the list".

**Comment by daniellekirkwood** [comment]
> For the latter, I would recommend telegram's design, which has a "go to the next mention" button above the "go to the bottom of the page" button.
> 
> (I could grab visuals for this on request)

Visuals would be great, thanks @ShadowJonathan

**Comment by daniellekirkwood** [comment]
> > (I could grab visuals for this on request)
> 
> probably vector-im/element-android#3822?
> 
> However my idea is instead of (supposedly, I don't have telegram) jumping to the last ping, give users agency to select them from a list instead of having to "click through the list".

Almost like a news feed? Instead of going to each room, you want a place you can go and read the highlights from all your places?

**Comment by ShadowJonathan** [comment]
> Visuals would be great, thanks @ShadowJonathan

It's more or less the first image of https://github.com/vector-im/element-android/issues/3822, but I'll repost it for the sake of brevity;

![](https://user-images.githubusercontent.com/41234183/129018030-2632d512-22e6-4d74-bc96-128b3efa0360.jpg)

**Comment by HarHarLinks** [comment]
> Almost like a news feed? Instead of going to each room, you want a place you can go and read the highlights from all your places?

yes, that is pretty much how it currently is on desktop. however i'd argue that you should also be able to filter this by limiting to room(s) and space(s) and if possible also more like only replies, only `@room`, only `@mxid`, by sender, ...

In fact I have requested the same for web: https://github.com/vector-im/element-web/issues/20308

**Comment by daniellekirkwood** [comment]
Great - I think this is a really great idea so I'm going to go ahead and move it to the Discussion section of our repo. 

I will combine any comments from the Web issue there too.

**Reply by AnisTaluqdar** [reply]
Yeah, I was always looking for this option. Actually, When we have long conversion or busy texting place we can not make track our own conversion and who replies and mention us. But telegram solve it easily. When you come back your group your all unread replies and mentions will be store in @ button you can easily reply one by one and you won't lose any replies and mentions this way. Recently, WhatsApp also working with this idea. Link: https://wabetainfo.com/whatsapp-messenger-beta-for-ios-22-1-71-whats-new/#more-11775

![WA_REPLIED_MENTIONED_NOTIFICATION_IOS-768x332 (1)](https://user-images.githubusercontent.com/41234183/148933292-c61392f9-4ca6-403f-8800-4df14c9f1ce1.png)

**Reply by AnisTaluqdar** [reply]
The way telegram works:

https://user-images.githubusercontent.com/41234183/148940188-4f17b497-d755-4eb1-86a4-326273b2ead2.mp4

@daniellekirkwood  You can watch the video how it works in telegram.

**Reply by HarHarLinks** [reply]
you showed what happens when there is one mention. what happens if there are more?

**Reply by AnisTaluqdar** [reply]
> you showed what happens when there is one mention. what happens if there are more?

That will be count in @ button and it will decreases if you read it.  Until you read it the reply or mentions count will be stay in @ button when you read it it will decreases. When you will read it all, the button will be hide.

**Comment by AnisTaluqdar** [comment]
@HarHarLinks for you.

https://user-images.githubusercontent.com/41234183/148944610-29b72bf7-7b0c-49e9-8ac5-5265ed47f118.mp4

**Reply by HarHarLinks** [reply]
so it always just goes to the newest(?) notification when you press the `@`. no way to select which one to go to, hm.

to be fair it's reasonably simple UX for mobile, perhaps even in general. still I would like to see it expanded to the feed like panel discussed earlier. maybe long pressing the button could take you there, among other things?
